plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/InMemoryThriftMetastore.java

```diff
@@ -205,7 +205,10 @@ public synchronized void createTable(HiveIdentity identity, Table table)
         }
 
         PrincipalPrivilegeSet privileges = table.getPrivileges();
-        if (privileges != null) {
+        if (privileges != null
+                && privileges.getUserPrivilegesSize() != 0
+                && privileges.getRolePrivilegesSize() != 0
+                && privileges.getGroupPrivilegesSize() != 0) {
```

Comment:
i guess you meant `||` not `&&` for these

Reply:
`if (privileges != null && !privileges.equals(new PrincipalPrivilegeSet()))`

Reply:
> i guess you meant || not && for these

Yes, by mistake. Fixing it